render.yaml

```
@@ -51,6 +51,11 @@ services:
         sync: false  # Set this in Render dashboard for security
       - key: PROKERALA_CLIENT_SECRET
         sync: false  # Set this in Render dashboard for security
+      # 🌍 Global Knowledge Collection Configuration  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)